uui-core/src/helpers/__tests__/numericInputCalculations.test.tsx

```diff
@@ -29,7 +35,6 @@ describe('numericInputCalculations', () => {
         expect(getMinMaxValidatedValue({ value: 10, min: 0, max: 10 })).toBe(10);
         expect(getMinMaxValidatedValue({ value: 12, min: 0, max: 10 })).toBe(10);
         expect(getMinMaxValidatedValue({ value: -1, max: 10 })).toBe(0);
-        expect(getMinMaxValidatedValue({ value: null, max: 10 })).toBe(null);
```

Comment:
why do we remove this case?

Reply:
ValidationDTO interface accept only `value: number`.